stand/boot/efi/libefi/libefi.c

```diff
@@ -48,6 +48,21 @@ efi_get_table(EFI_GUID *tbl)
 	return (NULL);
 }
 
+static void
```

Comment:
`static`? You won't be able to use it then.

Reply:
fixed